licenses/licenses_tests.py

```diff
@@ -460,3 +460,21 @@ def test__licenses__admin__LicenseRequestAdmin__5(browser, license_request):
     lr = LicenseRequest.objects.get(id=license_request.id)
     assert 'was changed successfully' in browser.contents
     assert license_request == lr
+
+
+def test__licenses__admin__LicenseRequestAdmin__6(browser):
+    """The number field is not visible when a LR gets add by an admin."""
+    browser.login_admin()
+    browser.follow('License Request')
+    browser.follow('Add License Request')
+
+    assert 'Number:' not in browser.contents
+
+
+def test__licenses__admin__LicenseRequestAdmin__7(browser, license_request):
+    """The number field is not visible when a LR gets add by an admin."""
```

Comment:
```suggestion
    """The number field is not visible if a LR is looked at by an admin."""
```
Maybe so? What does the number mean and why is it not allowed to see it?

Reply:
The number is kind of a second id that is visible to staff and admins. This is needed due to legacy data. The number of a new LR gets generated automatically, so the field shouldn't be visible to the staff/admin on creation, but it should be visible in the detail/edit view.